src/data/UserFunctions.js

```diff
@@ -111,16 +111,13 @@ export const addCommentFunc = comment => {
     .catch(err => err);
 };
 //Fetch  all comments
-export const allComments = () => {
-  return axios
-    .get(`/comments/getComments`)
-    .then(res => res)
-    .catch(err => err);
+export const allComments = async () => {
+  return await axios.get(`/comments/getComments`);
```

Comment:
Better just return axios.get, there's no use waiting here for it to resolve, especially if your'e not going to catch errors.